### PR TITLE
Remaking the samples

### PR DESCRIPTION
**Pull request by CharukaK**
## Purpose
some of the fixes for #3 

## Test environment
Node.JS v8.5.0, NPM v5.3.0

### CODE REVIEW

File: samples/App.jsx [hunk 1]
```diff
@@ -1,53 +1,15 @@
-/*
```
Comment (NisalaNiroshana) [inline samples/App.jsx]: Add the missing licence header here

File: samples/ChartWrapper.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```
Comment (NisalaNiroshana) [inline samples/ChartWrapper.jsx]: Lets add licence header here

File: samples/chart-docs/BarChartSamples.jsx [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (NisalaNiroshana) [inline samples/chart-docs/BarChartSamples.jsx]: Remove unnecessary license header. there are two license headers

File: samples/chart-docs/BarChartSamples.jsx [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+        this.barChartStackedConfig = {
+            x: 'rpm',
+            charts: [{ type: 'bar', y: 'torque', color: 'EngineType', colorDomain: ['', '', 'piston'], mode: 'stacked' }],
+            maxLength: 7,
+            legend: true,
+        };
+
+        this.barChartStackedConfigHorizontal = {
+            x: 'rpm',
+            charts: [{ type: 'bar', y: 'torque', color: 'EngineType', colorDomain: ['', '', 'piston'], mode: 'stacked', orientation: 'left' }],
```
Comment (NisalaNiroshana) [inline samples/chart-docs/BarChartSamples.jsx]: This line exceeds standard line length

File: samples/chart-docs/ScatterPlotSample.jsx [hunk 5]
```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (NisalaNiroshana) [inline samples/chart-docs/ScatterPlotSample.jsx]: Please change this to 2018

File: samples/chart-docs/PieChartSamples.jsx [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+export default class PieChartSamples extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            data: [[1, 10, 23, 'piston'], [1, 20, 34, 'rotary']],
+            data2: [[1, 10, 23, 'piston']],
+            timer: 0,
+        };
+
+        this.interval_jd = null;
```
Comment (lasanthaS) [inline samples/chart-docs/PieChartSamples.jsx]: Single convention pls `this.intervalId`

File: samples/chart-docs/ScatterPlotSample.jsx (hunk 5 above)
Comment (lasanthaS) [inline samples/chart-docs/ScatterPlotSample.jsx]: Fix the license header with the correct year.

File: samples/chart-docs/TableChartSamples.jsx [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+
+import React from 'react';
+import { AppBar, Toolbar, Typography, Grid, IconButton } from 'material-ui';
+import HomeIcon from 'material-ui-icons/ArrowBack';
+import { Link } from 'react-router-dom';
+import ChartWrapper from '../ChartWrapper';
+import VizG from '../../src/VizG';
+import { syntaxHighlight } from './helper';
+
+
```
Comment (lasanthaS) [inline samples/chart-docs/TableChartSamples.jsx]: Multiple newlines

File: samples/chart-docs/helper.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export function syntaxHighlight(json) {
```
Comment (lasanthaS) [inline samples/chart-docs/helper.js]: Better to have a meaningful filename (which indicates what's inside)

File: samples/ChartWrapper.jsx (hunk 2 above)
Comment (CharukaK) [reply samples/ChartWrapper.jsx]: done

File: src/components/ComponentGenerator.jsx [hunk 9]
```diff
@@ -1,4 +1,4 @@
-/*
+  /*
```
Comment (lasanthaS) [inline src/components/ComponentGenerator.jsx]: Spacing?

File: samples/App.jsx [hunk 10]
```diff
@@ -1,53 +1,15 @@
...
-            data: [[1, 10, 23, 'piston'], [1, 20, 34, 'rotary']],
-            data2: [[1, 10, 23, 'piston']],
-            scatterPlot: [
-                [1, 41600, 3.5, 79.91, 0.8, 'piston', 0.03],
-                [2, 37800, 3.5, 79.65, 1.3, 'rotary', 0.06]],
-            timer: 0,
+            data: [],
+            index: 0,
+            scatterPlot: [],
             mapData: [
```
[the remark this answers is not in the document]
Comment (CharukaK) [reply samples/App.jsx]: They are not constants they will be used in the JSX below to update the charts

[the remark this answers is not in the document]
Comment (lasanthaS) [reply samples/App.jsx]: Ack.

**Pull request merged**